packages/graphql/tests/integration/filtering/filter-interface-relationship-alias.int.test.ts

```diff
@@ -0,0 +1,331 @@
+
+describe("interface relationships aliased fields", () => {
+    const testHelper = new TestHelper();
+    const secret = "secret";
+
+    let typeMovie: UniqueType;
+    let typeSeries: UniqueType;
+    let typeActor: UniqueType;
+    let ProtectedActor: UniqueType;
+
+    beforeEach(async () => {
+        typeMovie = testHelper.createUniqueType("Movie");
+        typeSeries = testHelper.createUniqueType("Series");
+        typeActor = testHelper.createUniqueType("Actor");
+        ProtectedActor = testHelper.createUniqueType("ProtectedActor");
+
+        const typeDefs = /* GraphQL */ `
+            interface Production {
+                title: String!
+            }
+
+            type ${typeMovie} implements Production {
+                title: String! @alias(property: "movieTitle")
+                runtime: Int!
+            }
+
+            type ${typeSeries} implements Production {
+                title: String! @alias(property: "seriesTitle")
+                episodes: Int!
+            }
+
+            type ActedIn @relationshipProperties {
+                screenTime: Int!
+            }
+
+            type ${typeActor} {
+                name: String!
+                currentlyActingIn: Production @relationship(type: "CURRENTLY_ACTING_IN", direction: OUT)
+                actedIn: [Production!]! @relationship(type: "ACTED_IN", direction: OUT, properties: "ActedIn")
+            }
+
+            type ${ProtectedActor} @authorization(validate: [{ where: { node: { actedInConnection: { node: { title: "$jwt.title"  } } } } }]) {
+                name: String! @alias(property: "dbName")
+                actedIn: [Production!]! @relationship(type: "ACTED_IN", direction: OUT, properties: "ActedIn")
+            }
+        `;
+
+        await testHelper.initNeo4jGraphQL({ typeDefs, features: { authorization: { key: secret } } });
+    });
+
+    afterEach(async () => {
+        await testHelper.close();
+    });
+
+    test("should read and return interface relationship fields with interface relationship filter SOME", async () => {
+        const actorName = generate({
+            readable: true,
+            charset: "alphabetic",
+        });
+        const actorName2 = generate({
+            readable: true,
+            charset: "alphabetic",
+        });
+
+        const movieTitle = generate({
+            readable: true,
+            charset: "alphabetic",
+        });
+        const movieTitle2 = generate({
+            readable: true,
+            charset: "alphabetic",
+        });
+        const movieRuntime = faker.number.int({ max: 100000 });
+        const movieScreenTime = faker.number.int({ max: 100000 });
+
+        const seriesTitle = generate({
+            readable: true,
+            charset: "alphabetic",
+        });
+        const seriesEpisodes = faker.number.int({ max: 100000 });
+        const seriesScreenTime = faker.number.int({ max: 100000 });
+
+        const query = `
```

Comment:
You can have text highlighting by doing this:
```suggestion
        const query = /* GraphQL */ `
```